elements/designer-element-highlight/designer-element-highlight.html

```diff
@@ -0,0 +1,95 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html>
+  <head>
+    <link rel="import" href="../../../polymer/polymer.html">
+  </head>
+  <body>
+
+    <dom-module id="designer-element-highlight">
+      <style>
+        :host {
+          display: block;
+          position: absolute;
+          box-sizing: border-box;
+        }
+        :host.before {
+          border-top: solid 4px black;
+        }
+        :host.after {
+          border-bottom: solid 4px black;
+        }
+        #bounds {
+          position: absolute;
+          top: 0;
+          left: 0;
+          right: 0;
+          bottom: 0;
+          background: rgba(255, 128, 128, .5);
+          border: solid 1px #f88;
+          box-sizing: border-box;
+        }
+      </style>
+      <template>
+        <div id="bounds"></div>
+      </template>
+    </dom-module>
+
+    <script type="text/javascript">
+    define([
+        'polymer-designer/commands',
+        'polymer-designer/dom-utils'],
+      function(commands, domUtils) {
+        'use strict';
+
+        Polymer({
+          is: 'designer-element-highlight',
+
+          properties: {
+            bounds: Object,
+          },
+
+          show() {
+            this.style.display = 'block';
+          },
+
+          hide() {
+            this.style.display = 'none';
+          },
+
+          setBounds(bounds) {
+            this.bounds = bounds;
+            domUtils.setBounds(this, bounds);
+          },
+
+          showInsertMarker(x, y) {
+            let bounds = this.bounds;
+            let verticalMid = bounds.top + bounds.height / 2;
```

Comment:
Since `this.bounds` defaults to `undefined`, it seems possible for this to result in an attempt to access a property of `undefined`.